packs/zendesk.yml

```diff
@@ -11,9 +11,6 @@ PackDefinition:
     - Zendesk.UserAssumption
     - Zendesk.UserRoleChanged
     - Zendesk.UserSuspension
-    # Standard rules that apply to zendesk
```

Comment:
No action needed:

We opted to include Standard. rules only in the standard pack. This eases dependency resolution packaging. Additionally the standard rules will automagically act on only the enabled logtype for which they are configured to act upon.